lib/swoosh/adapters/mandrill.ex

```diff
@@ -47,10 +48,15 @@ defmodule Swoosh.Adapters.Mandrill do
 
   defp base_url(config), do: config[:base_url] || @base_url
 
+  defp api_endpoint(%{provider_options: %{template_name: _template_name}}), do: @template_api_endpoint
+  defp api_endpoint(%{provider_options: %{template_content: _template_content}}), do: @template_api_endpoint
```

Comment:
Reading the API docs it looks like `template_name` is mandatory if you want to use templates so we could use this to figure out which endpoint we need to use.